packages/material-ui-styles/src/withStyles.js

```diff
@@ -37,14 +39,15 @@ const withStyles = (stylesOrCreator, options = {}) => Component => {
   }
 
   const useStyles = makeStyles(stylesOrCreator, {
-    ...otherOptions,
+    defaultTheme,
     Component,
     name: name || Component.displayName,
     classNamePrefix,
+    ...stylesOptions,
   });
 
-  const WithStyles = React.forwardRef(function WithStyles(props, ref) {
-    const { classes: classesProp, innerRef, ...other } = props;
+  let WithStyles = React.forwardRef(function WithStyles(props, ref) {
+    const { classes: _, innerRef, ...other } = props;
```

Comment:
We don't usually do this. I'd rather stay consistent.

Reply:
OK